packages/cletus/src/common.ts

```diff
@@ -52,6 +52,62 @@ export function abbreviate(text: string, maxLength: number, suffix: string = '
   }
 }
 
+/**
+ * Format a value for text display (high-level representation without JSON escaping)
+ * - Arrays: items on separate lines with hyphens
+ * - Non-objects (primitives): String(x)
+ * - Objects: bullet list with hyphens, property values without JSON escaping
+ * 
+ * @param value - value to format
+ * @returns formatted string
+ */
+export function formatValue(value: any): string {
```

Comment:
Looks like you need to pass depth in here so the hyphenated lists are properly indented